packages/expo-cli/src/commands/eas-build/build/utils/gradleContent.ts

```diff
@@ -0,0 +1,52 @@
+export default `android {
```

Comment:
In my opinion, this is not a util. I'd move it outside the `utils` directory. Maybe create a directory named `templates` or similar and move the file there.